src/content/api/settlements.mdoc

```diff
@@ -55,6 +55,10 @@ Settlements can only be created if the Merchant has a [Settlement Config](/api/m
     The ID of the [Bank Account](/api/bank-accounts/) used for settlement.
   {% /property %}
 
+  {% property name="settlementBankTransferId" type="string" %}
+    The ID of the Bank Transfer used for settlement.
+  {% /property %}
```

Comment:
@Ray2126 
**Nitpick**  🧐
There is some inconsistency regarding the letter 'S' in the word "settlement": should it be capitalized or written in lowercase?
![Screenshot 2024-12-10 at 11 31 50 AM](https://github.com/user-attachments/assets/789efafe-ff40-4e61-bc67-758138c8fc7c)

Reply:
Nice catch, they should be capitalized as they are referring to a model. I'll fix this up